src/default_panels/StyleTracesPanel.js

```diff
@@ -43,12 +44,14 @@ import {
 const StyleTracesPanel = (props, {localize: _}) => (
   <TraceAccordion canGroup>
     <TextEditor label={_('Name')} attr="name" richTextOnly />
-    <ShowInLegend
-      label={_('Show in Legend')}
-      attr="showlegend"
-      options={[{label: _('Show'), value: true}, {label: _('Hide'), value: false}]}
-    />
     <NumericFraction label={_('Trace Opacity')} attr="opacity" />
+    <PlotlySection name={_('Show in Legend')}>
```

Comment:
Please rename section `Legend` and restore the `Show in Legend` label for the show/hide button.